utils/web_ui.py

```
@@ -34,6 +38,237 @@ def log_message(msg_type, message):
+                result = subprocess.run(
+                    command,
+                    shell=True,
+                    cwd=self.workspace_path,
+                    capture_output=True,
+                    text=True,
+                    timeout=30
+                )
+                output = result.stdout
+                if result.stderr:
+                    output += f"\nSTDERR:\n{result.stderr}"
+                return output
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)